DESCRIPTION

```diff
@@ -90,15 +90,20 @@ Imports:
     tools,
     utils,
     uuid (>= 0.1-4),
-    xml2 (>= 1.3.0)
+    xml2 (>= 1.3.0),
 Suggests:
     knitr,
     mime (>= 0.9),
     reader (>= 1.0.6),
     ritis (>= 1.0.0),
     taxize (<= 0.9.99),
     testthat,
-    worrms (>= 0.4.2)
+    worrms (>= 0.4.2),
+    scales,
+    ggrepel,
```

Comment:
Sounds like `ggrepel` addresses a scaling issue? Same recommendations given above for the `scales` package apply here.

Reply:
@sokole this should be a suggested package and any functions using it should support two modes of operation with/without `ggrepel`.